docs/source/modules/idealised_moist_phys.rst

```diff
@@ -0,0 +1,295 @@
++----------------------------+-----------------------------------------------------------------------------+---------+
+
+
+Upper Level Damping
+"""""""""""""""""""
+
+At upper levels, damping may be needed to account for subgrid-scale processes that decelerate fast upper-level winds. ``damping_driver_mod`` allows Rayleigh friction to be applied at upper levels, and is currently being configured to allow gravity wave drag options. This is switched on with:
+
++----------------------------+-----------------------------------------------------------------------------+---------+
+| Option                     | Summary                                                                     |Default  |
++============================+=============================================================================+=========+
+|``do_damping``              |If true, call ``damping_driver_mod``                                         |``False``|
++----------------------------+-----------------------------------------------------------------------------+---------+
+
+
+
+Land and hydrology
+^^^^^^^^^^^^^^^^^^
+
+Land and hydrology processes are predominantly dealt with in ``surface_flux_mod`` and ``mixed_layer_mod``, but land and bucket hydrology options are initialised here. We acknowledge that the bucket hydrology is adapted from code by (TS github), and follows (citation). Land and hydrology options in this module are:
+
++----------------------------+----------------------------------------------------------------------+-------------------+
+| Option                     | Summary                                                              |Default            |
++============================+======================================================================+===================+
+|``land_option``             |Selects how land-mask is defined, a summary of options is given below.|``False``          |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``land_file_name``          |Filename for the input land-mask.                                     |``'INPUT/land.nc'``|
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``land_field_name``         |Field name in the input land-mask netcdf.                             |``'land_mask'``    |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``bucket``                  |If true, use bucket hydrology.                                        |``False``          |
++----------------------------+----------------------------------------------------------------------+-------------------+
+|``init_bucket_depth``       |Value at which to initialise bucket water depth over ocean (large).   |``1000.``          |
```

Comment:
suggestion: Value at which to initialise bucket water depth over ocean (large, in :math: 'm').